static/libs/utils.js

```diff
@@ -77,6 +77,31 @@
     el.classList.add(c);
   };
 
+  Utils.addEvent = function(element, event, arr, func) {
```

Comment:
This is not generic enough to go to the `Utils.js` singleton.
Please listen to events in the standard JavaScript way

If you need a way to get the real target element that you need, take a look at `Utils.getRealTargetByClass` function:

`var realTargetEl = Utils.getRealTargetByClass(e.target, '.the-class');`